packages/pigment-css-unplugin/src/index.ts

```diff
@@ -347,7 +347,10 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
               if (id.endsWith('styles.css')) {
                 return theme ? generateTokenCss(theme) : _code;
               }
-              if (id.includes('pigment-css-react/theme')) {
+              if (
+                id.includes('pigment-css-react/theme') ||
```

Comment:
I wonder what's `pigment-css-react/theme` is for?

Reply:
At some point, it was for local testing in workspaces. We can maybe clean this up separately.